docs/user-guide/networkpolicies.md

```diff
@@ -0,0 +1,82 @@
+---
+---
+
+* TOC
+{:toc}
+
+## What is a _Network Policy_?
```

Comment:
Take this out; you define the title in /_data/reference.yml (suggest this goes under Glossary)